docs/how-to/configure.md

```diff
@@ -14,12 +14,12 @@ On the left are the selector tabs:
     - [Data Retention](#data-retention)
     - [Telemetry](#telemetry)
     - [Check for updates](#check-for-updates)
-    - [Advisors](#advisors)
+    - [Advisor checks](#advisor-checks)
```

Comment:
probably, name Advisors here too

Reply:
The UI for Advanced Settings uses "Advisor Checks"